string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,14 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  if (word.toLowerCase() === 'javascript') {
+    var firstLetterJava = word[0].toUpperCase();
+    var restOfJava = word.substring(1, 4).toLowerCase();
+    var jsScript = word[4].toUpperCase();
+    var restOfScript = word.substring(5).toLowerCase();
+    return firstLetterJava + restOfJava + jsScript + restOfScript;
```

Comment:
This is a very special case, so you don't need to assemble the return value programmatically, since you know what it needs to be already. You can just `return "JavaScript";`